### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by cshanjiewu** [MVIG-SJTU/pointSIFT#21]
where is the pre-trained model?

**Comment by yanx27** [comment]
Hello，I've trained the model by 200 epochs but didn't get result of 89% accuracy in your paper.  Could you give you pre-trained model, thx very much :)
